packages/enzyme-adapter-react-16/src/detectFiberTags.jsx

```diff
@@ -14,10 +15,38 @@ function getFiber(element) {
   return inst._reactInternalFiber.child;
 }
 
+function getLazyFiber(LazyComponent) {
+  const container = global.document.createElement('div');
```

Comment:
what happens when this runs on the server?

Reply:
`getFiber` (https://github.com/airbnb/enzyme/pull/1975/files#diff-bbd8c2174348cfcf3910cf8876ae1d7cR5) also use `global.document` so I think it's fine.